telemetry-all/README.md

```diff
@@ -0,0 +1,6 @@
+### Telemetry Combined Module
+
+This module provides high-level APIs for sending telemetry to New Relic.
```

Comment:
It's probably worth mentioning what this includes, and what it doesn't (for example, that it includes a java 11 http client sender)